### DISCUSSION TITLE
Next/Image responding 503 with configured next.config.js

### DISCUSSION THREAD

**Discussion by colinpcurtis** [vercel/next.js#44110] [Help]
I'm requesting an image using the following

```jsx
<Image src="https://miro.medium.com/max/1400/0*UBpNTmkhJo1LIjzz" alt='image' width={500} height={400} />
```

and this is what my `next.config.js` looks like

```js
/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  images: {
    formats: ['image/avif', 'image/webp'],
    remotePatterns: [
      {
        protocol: 'https',
        hostname: 'miro.medium.com',
        port: '',
        pathname: '/**'
      },
    ],
  },
}

module.exports = nextConfig
```
and every time I run the app I get the following error `upstream image response failed for https://miro.medium.com/max/1400/0*UBpNTmkhJo1LIjzz 503`

I'm really not too sure what could be happening since I think this is all configured correctly.

I'm perhaps thinking that the weird image url (`0*UBpNTmkhJo1LIjzz`) might be causing `next/Image` to error out.

The nextwork tab displays 503 on the request as well.

Any help would be greatly appreciated!

**Comment by ciruz** [comment]
Hey, which next.js version?

It works for me, in every browser - tested with next 13  (`next.js 13.0.6`) in ``/app`` folder.

Alternatively, you can just try the domain configuration https://nextjs.org/docs/api-reference/next/image#domains .

```javascript
const nextConfig = {
  reactStrictMode: true,
//  experimental: {
//    appDir: true,
//  },
  images: {
    domains: ['miro.medium.com'],
  },
}
```

If i modify the remotePattern to a different Domain, it works as expected and throws an error.

```
Uncaught Error: Invalid src prop (https://miro.medium.com/max/1400/0*UBpNTmkhJo1LIjzz) on `next/image`, hostname "miro.medium.com" is not configured under images in your `next.config.js`
```
